var/spack/repos/builtin/packages/homer/package.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from os import symlink
+
+
+class Homer(Package):
+    """Software for motif discovery and next generation sequencing analysis"""
+
+    homepage = "http://homer.ucsd.edu/homer"
+    url      = "http://homer.ucsd.edu/homer/data/software/homer.v4.9.zip"
+
+    version('4.9.1', sha256='ad1303b0b0400dc8a88dbeae1ee03a94631977b751a3d335326c4febf0eec3a9')
+
+    depends_on('perl', type=('build', 'run'))
+    depends_on('r-biocgenerics', type='run')
+    depends_on('r-biocparallel', type='run')
+    depends_on('r-edger', type='run')
+    depends_on('r-deseq2', type='run')
+
+    variant('data', default=False,
```

Comment:
As the 'data' and 'custom' variants are effectively exclusive, I would prefer a multi-valued variant, maybe: 
```suggestion
    variant('data', default='none', description='Which genome data packages to download',
            values=('none','all','minimal'))
```
With the matching logic in the `install()` below.